device-types/Supermicro/MBD-X8STi.yaml

```diff
@@ -0,0 +1,40 @@
+---
+manufacturer: Supermicro
+model: Motherboard X8STi
+slug: supermicro-mbd-x8sti
+part_number: MBD-X8STi
+u_height: 1
+is_full_depth: true
+comments: Supermicro X8STi motherboard - LGA1366, 2x1GbE, 6xDDR3, IPMI supported
+
+interfaces:
+  - name: eth0
+    type: 1000base-t
+    mgmt_only: false
+  - name: eth1
+    type: 1000base-t
+    mgmt_only: false
+power-ports:
+  - name: PSU1
+    type: iec-60320-c14
+device-bays: []
+module-bays: []
+inventory-items:
```

Comment:
Inventory items are being phased out so not something we would be looking to add